static/js/legend.js

```diff
@@ -150,6 +153,40 @@ function setAllMapValues(map,id) {
         tags.forEach(tag => useTypes.set(tag, map==null?false:true));
     }
 }
+
+/*
+Name: checkSelectAll
+
+Usage: selecting/deselecting all checkboxes of either Use Type or States will select/deselect the corresponding select all checkbox
+*/
+function checkSelectAll(){
+    //grabs select all checkboxes by HTML id
+    const statesBox = document.getElementById("States-checkbox");
+    const typeBox = document.getElementById("types-checkbox");
+    
+    //Set to filter in/out states & types
+    const excludeStates = new Set(['Maryland', 'Virginia']);
+    const excludeTypes = new Set(['Agriculture', 'Aquaculture','Commercial','Fossil Power', 'Industrial','Irrigation','Manufacturing','Mining','Municipal','Nuclear Power','Other']);
```

Comment:
This is redundant since arrays have already been created containing this information.

Reply:
removed the redundancy